tools/packaging/common/envoy_bootstrap_v2.json

```diff
@@ -579,6 +594,14 @@
                             "route": {
                               "cluster": "prometheus_stats"
                             }
+                          },
```

Comment:
nit: by reusing the http connection manager, it reuses the stat prefix which is "stats" - May not be a big deal, but just pointing this out.

Reply:
Is it worth the overhead of making 2 HCMs? Or should we make stat_prefix more generic?

Reply:
> Is it worth the overhead of making 2 HCMs

It is not a big problem so not worth

> Or should we make stat_prefix more generic?

We could. I thought about that but that needs updating stat prefixes in envoy config which is easy to do. But any dashboard level changes might be a problem? Not sure where this is used.  If folks already use these stats some where, they will be incorrect with this. If it is not used any where, I think we should leave it as is.